crypto/odocrypt.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2019 The DigiByte Developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef HASH_ODO
+#define HASH_ODO
+
+#include <stdlib.h>
+
+#define ODOCRYPT_CHAPECHANGE_INTERVAL (10*24*60*60)
+#define ODOCRYPT_DIGEST_SIZE          (80)
+
+// Number of rounds.
+const static int ROUNDS = 84;
```

Comment:
It would be slightly better to add a prefix to all of the constants.

Reply:
Will do that in some hours 👍